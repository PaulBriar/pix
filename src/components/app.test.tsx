import React from 'react';
import { shallow } from 'enzyme';

import App from './app';

describe('App Component', () => {
    it('Should render App component', () => {
        const component = shallow(<App/>);
        expect(component).toMatchSnapshot();
    });
});