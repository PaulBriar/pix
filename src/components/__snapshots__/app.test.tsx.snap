// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component Should render App component 1`] = `
<div
  className="ui container"
>
  <SearchBar />
</div>
`;
